packages/components/ContextualMenu/src/ContextualMenuItem.settings.ts

```diff
@@ -0,0 +1,66 @@
+import { contextualMenuItemName, IContextualMenuItemType } from './ContextualMenuItem.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+
+export const settings: IComposeSettings<IContextualMenuItemType> = [
+  {
+    tokens: {
+      backgroundColor: 'buttonBackground',
+      color: 'buttonText',
+      borderColor: 'buttonBorder',
+      borderWidth: 1,
```

Comment:
So I think these are reasonable, but I am curious whether talking to PM and design you felt like there was a possibility that things like borderWidth and borderRadius should be theme lookups instead. I could imagine some UIs having themes where these values need to change.

Reply:
I agree with you. I'll remove borderWidth and borderRadius from settings